libs/application/template-api-modules/src/lib/modules/templates/passport/passport.service.ts

```diff
@@ -37,6 +37,20 @@ export class PassportService extends BaseTemplateApiService {
     return identityDocument
   }
 
+  async deliveryAddress({ auth }: TemplateApiModuleActionProps) {
+    const res = await this.passportApi.getDeliveryAddress(auth)
+    if (!res) {
+      throw new TemplateApiError(
+        {
+          title: coreErrorMessages.failedDataProvider,
+          summary: coreErrorMessages.errorDataProvider,
+        },
+        400,
```

Comment:
Would this be easier to trace with `res.status` if that's available to us, instead of putting down 400 explicitly?